astropy/modeling/core.py

```diff
@@ -72,7 +72,7 @@ def _opfunc(left, right):
         # Deprecation is for https://github.com/astropy/astropy/issues/8234
         if not (isinstance(left, Model) and isinstance(right, Model)):
             warnings.warn(
-                'Composition of model classes will be removed in 4.0'
+                'Composition of model classes will be removed in 4.0 '
```

Comment:
Trailing whitespace unnecessary.
```suggestion
                'Composition of model classes will be removed in 4.0'
```

Reply:
it is otherwise there is no space between this and text in next line

Reply:
Ohh... okay. Don't mind me then. Sorry for the noise. LGTM, except I don't know if Tom R wanna also address https://github.com/astropy/astropy/pull/8736#issuecomment-494119542 here or not.

Reply:
I suppose it makes sense to remove the narrative docs of a deprecated behaviour at the same time as the deprecation, otherwise it can be still picked up and used.